app/views/exam_templates/_student_info.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= form_for [@assignment, exam_template],
+             url: add_fields_assignment_exam_template_path(id: exam_template.id) do |f| %>
+
+  <p>
+    <%= check_box_tag :automatic_parsing, true,
+                      exam_template.automatic_parsing,
+                      onchange: "toggle_cover_page(#{exam_template.id}, #{exam_template.cover_fields.split(',')}); return true;" %>
+    Automatically parse student information<br>
+  </p>
+
+  <div id="exam-cover-display-<%= exam_template.id %>" style="display: <% if exam_template.automatic_parsing %>flex<% else %>none<% end %>;">
+    <div id="exam_cover-<%= exam_template.id %>">
+      <img src=<%= show_cover_assignment_exam_template_path(id: exam_template.id) %> id="crop-target" style="max-width: 400px; border: 1px solid black;">
+    </div>
+
+    <% if exam_template.automatic_parsing %>
+      <script>attach_crop_box()</script>
+    <% end %>
+
+    <div class="division-table" style="padding:50px;">
+      <header>
+        <span class="label"><%= "Field" %></span>
+        <span class="start"><%= "Value" %></span>
+      </header>
+      <ul id="cover-fields-<%= exam_template.id %>">
+        <% for i in 1..4 do %>
+          <li>
+            <span><%= i %></span>
+            <%= select_tag("field#{i}", options_for_select({" " => " ",
+                                                            "Full Name" => "full_name",
+                                                            "First Name" => "first_name",
+                                                            "Last Name" => "last_name",
+                                                            "Student Number/ID" => "student_id",
+                                                            "Username" => "username"},
+                                                           exam_template.get_cover_field(i-1))) %>
+          </li>
+        <% end %>
+        <li>
+          <%= hidden_field_tag 'x', exam_template.crop_x %>
```

Comment:
You don't need a separately `li` for this (in fact, these hidden fields don't need to be inside the `ul` at all).